packages/aws-cdk-lib/aws-ec2/lib/private/ebs-util.ts

```diff
@@ -25,7 +25,30 @@ function synthesizeBlockDeviceMappings<RT, NDT>(construct: Construct, blockDevic
 
     if (ebs) {
 
-      const { iops, volumeType, kmsKey, ...rest } = ebs;
+      const { iops, throughput, volumeType, kmsKey, ...rest } = ebs;
+
+      if (throughput) {
+        const throughputRange = { Min: 125, Max: 1000 };
+        const { Min, Max } = throughputRange;
+
+        if (volumeType != EbsDeviceVolumeType.GP3) {
+          throw new Error('throughput property requires volumeType: EbsDeviceVolumeType.GP3');
+        }
+
+        if (throughput < Min || throughput > Max) {
+          throw new Error(
+            `throughput property takes a minimum of ${Min} and a maximum of ${Max}`,
+          );
+        }
```

Comment:
```suggestion
        if (volumeType != EbsDeviceVolumeType.GP3) {
          throw new Error(`throughput requires volumeType: ${EbsDeviceVolumeType.GP3}, got: ${volumeType}.`);
        }

        if (throughput < 125 || throughput > 1000) {
          throw new Error(`throughput must be between 125 and 1000, got ${throughput}.`);
        }
```
Let's keep it cleaner and provide more feedback on the error message.